### PR TITLE
Add React footer to app

### PR DESCRIPTION
**Pull request by ProbablyFaiz**

### CODE REVIEW

**Comment by ProbablyFaiz** [comment]
[React Issue 1: Add Footer](https://trello.com/c/MheLjvq9/1-react-issue-1-add-footer)

File: app/javascript/image_sharing/components/App.js [hunk 1]
```diff
@@ -1,20 +1,16 @@
 import React, { Component } from 'react';
 import { inject } from 'mobx-react';
-import Header from './Header';
-import Footer from "./Footer";
+import Footer from './Footer';
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/App.js:3]: I dont think you should remove the header

File: app/javascript/image_sharing/components/Footer.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import React, { Component } from 'react';
 
 class Footer extends Component {
-  /* Implement your Footer component here */
+  render() {
+    return (
+      <footer>
+        <div className="footer-text" style={{ fontSize: '10px', textAlign: 'center' }}>
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/Footer.js:7]: you can add like a custom className here and directly find it in the test.

File: .eslintrc.json [hunk 3]
```diff
@@ -9,6 +9,8 @@
   "settings": {
     "import/resolver": {
       "webpack": {}
-    }
+    },
+    "import/no-unresolved": "off",
```
Comment (bhargav265) [inline .eslintrc.json:13]: Can you make a new commit for fixing lint errors?

File: app/javascript/image_sharing/components/Footer.js (hunk 2 above)
Comment (ProbablyFaiz) [reply app/javascript/image_sharing/components/Footer.js:7]: That's what class ``footer-text`` does. It's tested in the spec file

File: .eslintrc.json (hunk 3 above)
Comment (ProbablyFaiz) [reply .eslintrc.json:13]: Done

File: app/javascript/image_sharing/components/App.js (hunk 1 above)
Comment (ProbablyFaiz) [reply app/javascript/image_sharing/components/App.js]: Changed

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -7,4 +7,4 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-import '../image_sharing'
+import '../image_sharing';
```
Comment (bhargav265) [inline app/javascript/packs/application.js:10]: I meant to say to put these lint errors in the next commit too.

File: app/javascript/image_sharing/utils/helper.js [hunk 5]
```diff
@@ -16,12 +17,12 @@ function getCsrfToken() {
  */
 export function serialize(obj, prefix) {
   const parts = [];
-  Object.keys(obj).forEach(key => {
+  Object.keys(obj).forEach((key) => {
```
Comment (bhargav265) [inline app/javascript/image_sharing/utils/helper.js:20]: this too

File: app/javascript/image_sharing/services/PostFeedbackService.js [hunk 6]
```diff
@@ -1,4 +1,5 @@
-import { post } from "../utils/helper";
+// eslint-disable-next-line no-unused-vars
```
Comment (bhargav265) [inline app/javascript/image_sharing/services/PostFeedbackService.js:1]: this one

File: app/javascript/image_sharing/components/Header.js [hunk 7]
```diff
@@ -19,7 +19,7 @@ class Header extends Component {
           </Col>
         </Row>
       </div>
-    )
+    );
```
Comment (bhargav265) [inline app/javascript/image_sharing/components/Header.js:22]: this one

File: app/controllers/api/feedbacks_controller.rb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 module Api
   class FeedbacksController < ApplicationController
     def create
-     # Implement your create action
+      # Implement your create action
```
Comment (bhargav265) [inline app/controllers/api/feedbacks_controller.rb:4]: this one

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -16,6 +16,7 @@
   <body>
     <div class="main-content">
       <%= yield %>
+      <div id="footer-root"></div>
```
Comment (bhargav265) [inline app/views/layouts/application.html.erb:19]: can you add a test for this?

**Pull request merged**